codespell_lib/data/dictionary.txt

```diff
@@ -12899,6 +12899,11 @@ servce->service
 servces->services
 serveice->service
 serveral->several
+serverite->severity
+serverites->severities
+serveritie->severity,severities,
```

Comment:
"AssertionError: error serveritie: correction 'severity,severities,' contains a comma *not* followed by a space"

Reply:
Thanks, completely missed this.